content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -124,7 +124,7 @@ The following table lists common MongoDB features and describes the level of sup
 
 | Feature                                   | Supported by Prisma | Notes                                                                                                                                                                                                         |
 | :---------------------------------------- | :------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Embedded documents                        | Not formally        | Embedded documents are not formally supported, but you can technically use the `Json` field to manage embedded documents on your own. See [GitHub issue #6708](https://github.com/prisma/prisma/issues/6708). |
+| Embedded documents                        | Work in progress    | Embedded documents are not yet supported, but you can use the `Json` field while we work on proper support. See [GitHub issue #8272](https://github.com/prisma/prisma/issues/8272) to minitor progress. |
 | Transactions                              | ✔️                  |                                                                                                                                                                                                               |
```

Comment:
--"monitor"
-- consider changing ".. we work on proper support" to "... we work to provide support".

Reply:
fixed